### PR TITLE
Features/guided tour

### PR DESCRIPTION
**Pull request by soupette**
Create the guided tour plugin.

### CODE REVIEW

File: api/plugins/guided-tour/admin/src/components/Tour/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    dispatch({ type: 'TOGGLE_IS_OPEN', pluginId });
+  }, []);
+
+  const steps = useMemo(() => {
+    return Object.values(tour).reduce((acc, current) => {
+      return [...acc, ...current.steps];
+    }, []);
+  }, [tour]);
+
+  useEffect(() => {
```
Comment (soupette) [inline api/plugins/guided-tour/admin/src/components/Tour/index.js:27]: This effect will be run on every render, can you add the `tour, pluginId, actualPlugin` to the dependency array?

Comment (Mcastres) [reply api/plugins/guided-tour/admin/src/components/Tour/index.js:27]: Done

**Review by soupette — COMMENTED** [review]
LGTM

**Review by Mcastres — APPROVED** [review]
LGTM

**Pull request merged**